### PR TITLE
@ashfurrow=>Logout process + settings menu view model

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] Created a view model for the Settings View (using the friendlier term 'Menu' rather than 'Master')
- [x] Added logout functionality
- [x] Tests

### CODE REVIEW

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m [hunk 1]
```diff
@@ -29,35 +36,38 @@ @implementation ARLabSettingsMasterViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
+    
+    _viewModel = _viewModel ?: [[ARLabSettingsMenuViewModel alloc] init];
```
Comment (ashfurrow) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: :tada:

File: Classes/Controllers/Settings/ARLabSettingsMenuViewModel.m [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            return NSLocalizedString(@"Edit Presentation Mode", @"Title for edit presentation mode settings button");
+        case ARLabSettingsSectionBackground:
+            return NSLocalizedString(@"Background", @"Title for background settings button");
+        case ARLabSettingsSectionEmail:
+            return NSLocalizedString(@"Email", @"Title for email settings button");
+        case ARLabSettingsSectionSupport:
+            return NSLocalizedString(@"Support", @"Title for support button");
+        case ARLabSettingsSectionLogout:
+            return NSLocalizedString(@"Logout", @"Title for logout button");
+        default:
```
Comment (ashfurrow) [inline Classes/Controllers/Settings/ARLabSettingsMenuViewModel.m]: The `switch` looks like it covers all the cases of the enum, do we need a `default:` case?

File: Classes/Controllers/Settings/ARLabSettingsMenuViewModel.m [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+- (NSAttributedString *)presentationModeExplanatoryText
+{
+    return [NSLocalizedString(@"Hides sensitive information when showing artworks to clients", @"Explanatory text for presentation mode setting") attributedStringWithLineSpacing:5];
+}
+
+- (BOOL)presentationModeOn
+{
+    return [self.defaults boolForKey:ARPresentationModeOn];
+}
+//
```
Comment (ashfurrow) [inline Classes/Controllers/Settings/ARLabSettingsMenuViewModel.m]: `//` ?

**Comment by ashfurrow** [comment]
Fab, looks great! I usually put static text for UI like `confirmLogoutText` in the view controller (or storyboard!) but that's just a stylistic choice. Well done – feel free to merge when you're ready!

**Comment by sarahscott** [comment]
:dancer:  thanks @ashfurrow !

File: Classes/Controllers/Settings/ARLabSettingsMenuViewModel.h [hunk 4]
```diff
@@ -0,0 +1,24 @@
+#import "ARAppDelegate.h"
+#import "ARLabSettingsSplitViewController.h"
+
+
+@interface ARLabSettingsMenuViewModel : NSObject
```
Comment (orta) [inline Classes/Controllers/Settings/ARLabSettingsMenuViewModel.h]: would be awesome to add some `///` comments to this pleasssseeeee

**Pull request merged**